values.yaml

```diff
@@ -34,7 +38,7 @@ global:
   # Note: support for the catalog sync's liveness and readiness probes was added
   # to consul-k8s 0.6.0. If using an older consul-k8s version, you may need to
   # remove these checks to make the sync work.
-  # If using mesh gateways and bootstrapACLs then must be >= 0.9.0.
+  # If using bootstrapACLs then must be >= 0.10.1.
```

Comment:
Does it have to be? it's only if you're overriding the name, right? Also, should we keep the note about mesh gateways?

Reply:
It has to be because we're setting the new flags which won't work with the older consul-k8s

Reply:
I removed mesh gateways because previously it was only if you were using both, not it's if you're using acls at all which is inclusive of acls and mesh gateways